externs/browser/w3c_event.js

```diff
@@ -134,6 +134,12 @@ Event.prototype.timeStamp;
 Event.prototype.path;
 
 /**
+ * Present for events spawned in browsers that support shadow dom.
+ * @type {function():Array<!EventTarget>|undefined}
```

Comment:
Please add a @see tag and include a URL to the section of the specs that describes this.